packages/profile-sidebar/reducer.js

```diff
@@ -267,6 +282,9 @@ export default (state = initialState, action) => {
         isOnBusinessTrial: action.result.isOnBusinessTrial,
         userId: action.result.id,
         isFreeUser: action.result.is_free_user,
+        isOrganizationSwitcherEnabled: isOrgSwitcherFeatureEnabled(
+          action.result
+        ),
```

Comment:
This is just a suggestion 🙂 Since the feature is temporary and the code is not gonna be reutilised, I usually prefer to keep it small and without additional functions so it's easier to clean
```suggestion
        isOrganizationSwitcherEnabled: !!action.result.features?.includes('org_switcher'),
```

Reply:
That makes sense, I'll change that :+1: :slightly_smiling_face: